<ui version="4.0" >
 <class>ClientPrinterForm</class>
 <widget class="QWidget" name="ClientPrinterForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Print</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Symbolon</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QCheckBox" name="symbolon_ascsun_check" >
          <property name="text" >
           <string>Asc + Sun</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolon_dialectic_check" >
          <property name="text" >
           <string>Dialectic</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolon_ascinfluence_check" >
          <property name="text" >
           <string>Asc influence</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolon_suninfluence_check" >
          <property name="text" >
           <string>Sun influence</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolon_mcinfluence_check" >
          <property name="text" >
           <string>MC influence</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolon_housemandala_check" >
          <property name="text" >
           <string>House Mandala</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolon_planetsinhouses_check" >
          <property name="text" >
           <string>Planets in Houses</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolon_planetsinsigns_check" >
          <property name="text" >
           <string>Planets in signs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolon_planetanalysis_check" >
          <property name="text" >
           <string>Planet analysis (10 page)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="symbolon_planetaspects_check" >
          <property name="text" >
           <string>Planet aspects (10 page)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Classic</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="classic_circular_check" >
            <property name="text" >
             <string>Circular Scope</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="classic_rectangular_check" >
            <property name="text" >
             <string>Rectangular Scope</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="classic_box_check" >
            <property name="text" >
             <string>Box Style Scope</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Classic data</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="classicdata_basic_check" >
            <property name="text" >
             <string>Basic Datas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="classicdata_aspectmatrix_check" >
            <property name="text" >
             <string>Aspect Matrix</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="classicdata_qulaitymatrices_check" >
            <property name="text" >
             <string>Quality Matrices</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="classicdata_planetinfo_check" >
            <property name="text" >
             <string>Planet Info (10 page)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="print_button" >
       <property name="text" >
        <string>Print</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
